src/entity/column.rs

```diff
@@ -338,6 +340,42 @@ pub trait ColumnTrait: IdenStatic + Iterable + FromStr {
     fn into_expr(self) -> Expr {
         Expr::expr(self.into_simple_expr())
     }
+
+    /// Cast column expression used in select statement.
+    /// By default it only cast database enum as text.
```

Comment:
What I don't understand is, under what circumstance is not `default`?
How do I customize the behaviour?
This seems to unfortunately be part of the public API, and is it intended? Can we hide it instead?

Reply:
It's intended to be part of the public API. For example, user can override it and cast `citext` column type as `text` during select, `SELECT CAST("citext_column" AS text), ...`.

Reply:
I'll add a self-explanatory test case now

Reply:
Added https://github.com/SeaQL/sea-orm/pull/1304/commits/6295fcf185532033b8306e179a4e396cb5526461